src/lib/tooltip/tooltip.spec.ts

```diff
@@ -76,6 +90,43 @@ describe('MdTooltip', () => {
       expect(tooltipDirective._tooltipInstance).toBeNull();
     }));
 
+    it('should show and hide the tooltip if changeDetection is OnPush', fakeAsync(() => {
```

Comment:
Can you move this test to a separate describe block called "with OnPush" as a sibling to the existing "basic usage" block? Then you can do the `createComponent` in the `beforeEach`.